.github/workflows/ansible-test.yml

```
@@ -86,36 +87,76 @@ jobs:
           - 3.7
           - 3.8
           - 3.9
+        vault:
+          - 1.6.0
+          - 1.5.5
+          - 1.4.7
         exclude:
           - ansible: stable-2.9
             python: 3.9
 
     steps:
+      - name: Initialize env vars
+        run: |
+          COLLECTION_PATH=ansible_collections/${NAMESPACE}/${COLLECTION_NAME}
+          LOOKUP_HASHI_VAULT_PATH=${COLLECTION_PATH}/tests/integration/targets/lookup_hashi_vault
+          LOOKUP_HASHI_VAULT_BIN=${LOOKUP_HASHI_VAULT_PATH}/lookup_hashi_vault/files/bin
+          LOOKUP_HASHI_VAULT_VARS=${LOOKUP_HASHI_VAULT_PATH}/lookup_hashi_vault/vars
+
+          echo "COLLECTION_PATH=${COLLECTION_PATH}" >> $GITHUB_ENV
```

```suggestion
          echo "COLLECTION_PATH=${COLLECTION_PATH}" >> ${GITHUB_ENV}
```
(same for the next ones?)